docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
We've started setting and using "description" so this makes a lot of sense to us.


> On Jan 16, 2015, at 2:33 AM, bgrant0607 <notifications@github.com> wrote:
> 
> In docs/cli-usage.md:
> 
> > @@ -0,0 +1,96 @@
> > +## Using Kubernetes from the command line
> Something else we discussed somewhere was descriptions: --description="This is my foo server for bonking bobbles." should add a description annotation. We could display descriptions in the UI.
> 
> I'll pull this discussion into a cli/config roadmap (#2235).
> 
> —
> Reply to this email directly or view it on GitHub.
>